html/bleaching.py

```diff
@@ -0,0 +1,164 @@
+import bleach
+
+
+class Cleaner(bleach.Cleaner):  # pylint: disable=too-few-public-methods
+    """
+    This class will only help lowering the strictness level of `bleach.Cleaner` super class by redefining the safe
+    values we're currently using and the prohibited ones.
+    """
+    def __init__(self, strict=True):
+        """
+        This is an override to the original bleaching cleaner, it deals with two bleaching modes, the strict mode,
+        and the trusted mode.
+
+        - Strict Mode (default): Mainly used for rendering strings that are not required to have too much attributes
+          and tags and doesn't aloow the use of most of the styles attributes.
+        - Trusted mode: Mainly used to render content that marked as trusted.
+
+        :param strict: whether we should strictly process the text or not.
+        """
+        attributes = {
+            'a': ['href', 'title', 'target', 'rel'],
+            'span': ['style'],
```

Comment:
Do you need to pass `style` as an allowed attribute? From the docs it looked like it always allowed `style`, but it filters them based on the list of allowed styles.

Reply:
Yes, it's needed. I tested the code without them and every things got bleached when these attributes are missing. I checked the source code of the bleacher and the only allowed attributes are:
```python
ALLOWED_ATTRIBUTES = {
    'a': ['href', 'title'],
    'abbr': ['title'],
    'acronym': ['title'],
}
```